docs/topics/settings.rst

```diff
@@ -604,6 +604,26 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
+-----------------------
+
+Default: ``False``
+
+Whether or not to allow broken responses, where declared ``Content-Length``
+does not match content sent by the server. If ``False``, these responses
+trigger a ``ResponseFailed`` error. If ``True``, these responses are allowed
+and the flag ``partial`` is added.
```

Comment:
The flag is not exclusive to this condition though. Legit ``PotentialDataLoss`` responses which were not dropped have the ``partial`` flag. I was wondering if this condition ``_DataLoss`` should have its own flag, like ``data-loss``.